src/js/components/Form.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import uuidvl from "uuid";
+import { addArticle } from "../actions/index";
+
+const mapDispatchToProps = dispatch => {
+  return {
+    addArticle: article => dispatch(addArticle(article))
+  };
+};
+
+class ConnectedForm extends Component {
+  constructor() {
```

Comment:
you can write follow this way for a shorter code and not need constructor anymore:
state = {
  title: ""
}

handleChange = (event) => {
}

....

Reply:
fixed.